### PR TITLE
HSEARCH-2528 Document the dynamic mapping feature

### PR DESCRIPTION
**Pull request by yrodiere**
https://hibernate.atlassian.net//browse/HSEARCH-2528

### CODE REVIEW

**Comment by yrodiere** [comment]
Jenkins, retest this please

File: documentation/src/main/asciidoc/elasticsearch-integration.asciidoc [hunk 1]
```diff
@@ -757,6 +751,63 @@ This interface and `FieldBridge` in general are likely going to evolve in the ne
...
+}
+--
+
+[NOTE]
+---
+Dynamic mapping involves Elasticsearch guessing the type and attributes of fields automatically,
+and has its own limitations.
+See https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-mapping.html[the reference documentation]
+for more information. 
+---
```
Comment (Sanne) [inline documentation/src/main/asciidoc/elasticsearch-integration.asciidoc]: something is wrong here, it's not rendering the note but renders the `---` literally..

Comment (Sanne) [inline documentation/src/main/asciidoc/elasticsearch-integration.asciidoc]: I'll fix it, no need to repeat the PR dances...

**Comment by Sanne** [comment]
merged! Thanks. Fixed only some details in the note.

File: documentation/src/main/asciidoc/elasticsearch-integration.asciidoc (hunk 1 above)
Comment (yrodiere) [inline documentation/src/main/asciidoc/elasticsearch-integration.asciidoc]: Thanks! So it seems I should use two hyphens, not three. I'll try to remember.

**Pull request closed without merging**